docs/requisitos/elicitacao/brainstorming.md

```diff
@@ -6,6 +6,7 @@
 |---|---|---|---|
 | 12/04/2019 | 1.0 | Primeira versão do documento| Gabriela Guedes|
 | 13/04/2019 | 1.1 | Adicionando Informações do Brainstorming Realizado | Alexandre Miguel |
+|05/05/2019| 1.2 | Incluindo referências | Letícia Meneses |
 
 ## Introdução
```

Comment:
It would be better to have an introduction that describes what is a brainstorming